### ISSUE TITLE
Project proposal topic adjustment

### ISSUE THREAD

**Issue by josemoscoso-duke** [MIDS-at-Duke/unifying-data-science-final-project-pandemics-unemployment#1]
@MIDS-at-Duke/financial-crisis

Following on Nick's feedback on our project proposal draft, I have been looking for more detailed information about diverse virus impact in America's region with data from the Panamerican Health Organization.

https://www.paho.org/data/index.php/es/temas/indicadores-dengue/dengue-nacional/9-dengue-pais-ano.html

I have been following some specific studies that tried to evaluate the correlation of poverty and specific virus (dengue or zika for example). We could evaluate specific financial events those years, on poverty and any of those endemic problems. We can check the following study as a guideline.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4445289/

I will use today's office hours to review this, and also request another time slot that allows all team members to participate.

Feedback:
"So... I like the topic a lot, but I think it'll be REALLY hard to isolate the effect you're looking for. You're looking across countries (which often vary for many reasons), over long periods of time (so lots of things can happen during the window of the study), and Trade Balances and GDPs move around for all sorts of reasons (i.e. in other words, absent an epidemic, it's unlikely different regions would have parallel economic trajectories...). Plus there'd probably be some SUTVA violations (the treatment status of one country has effects on other countries assigned to control). So it's not clear you'd ever find a set of countries that would give you parallel trends pre-treatment. So I'd suggest trying to measure effects on a smaller scale. For example, there are ways to measure economic activity in China in real-time: https://www.theguardian.com/environment/2020/mar/23/coronavirus-pandemic-leading-to-huge-drop-in-air-pollution You could look at how that relates to measures taken in china? Or you could look at US quarantine measures and unemployment filings?"

**Comment by abbarcenasj** [comment]
UPDATE:

We changed our topic to the effect of stay-at-home orders on unemployment. We will measure unemployment by means of weekly unemployment claims reported by the US Department of Labor. 

We decided to analyze two approaches. The first approach is to measure the effect nationwide using a pre-post analysis and a fixed-effects linear regression. The second approach is to apply a difference-in-difference methodology at a state level.

More details on the motivation, data, and project design can be found on our README file. @nickeubank
